packages/runtimes/js/src/helpers/browser/js-loader.js

```diff
@@ -18,10 +18,13 @@ module.exports = cacheLoader(function loadJSBundle(bundle) {
     script.type = 'text/javascript';
     script.charset = 'utf-8';
     script.src = bundle;
-    script.onerror = function (e) {
+    script.onerror = function(e) {
+      var error = new TypeError(
```

Comment:
This should probably just be `new Error`. I don't think TypeError makes sense here

Reply:
from the chrome devtools:
```
> import("http://google.com")
Uncaught (in promise) TypeError: Failed to fetch dynamically imported module: http://google.com/
```

I don't think `TypeError` is specced somewhere though because the behaviour is implementation defined

https://tc39.es/ecma262/#sec-import-call-runtime-semantics-evaluation
https://tc39.es/ecma262/#sec-hostimportmoduledynamically

Reply:
Do you think we can keep `TypeError` to be inline with browsers behaviour though?